### ISSUE TITLE
Unable to load PowerShell

### ISSUE THREAD

**Issue by toniino38** [PowerShell/vscode-powershell#1835]
Hello,

I am facing an issue with PowerShell.

Here my logs (verbose mode) : 

```
02/04/2019 09:19:45 [NORMAL] - Path specified by 'powerShellExePath' setting - '' - not found, reverting to default PowerShell path.
02/04/2019 09:19:45 [NORMAL] - Language server starting --
02/04/2019 09:19:45 [NORMAL] -     args: C:\Users\XXX\.vscode\extensions\ms-vscode.powershell-1.12.0\modules\PowerShellEditorServices\Start-EditorServices.ps1 -HostName 'Visual Studio Code Host' -HostProfileId 'Microsoft.VSCode' -HostVersion '1.12.0' -AdditionalModules @('PowerShellEditorServices.VSCode') -BundledModulesPath 'C:\Users\XXX\.vscode\extensions\ms-vscode.powershell-1.12.0\modules' -EnableConsoleRepl -LogLevel 'Verbose' -LogPath 'C:\Users\XXX\.vscode\extensions\ms-vscode.powershell-1.12.0\logs\1554189585-9fb7402d-29cb-486d-a08d-f0ba3cdfc77d1554189581112\EditorServices.log' -SessionDetailsPath 'C:\Users\XXX\.vscode\extensions\ms-vscode.powershell-1.12.0\sessions\PSES-VSCode-12272-277559' -FeatureFlags @()
02/04/2019 09:19:45 [NORMAL] -     exe: C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe
02/04/2019 09:19:47 [NORMAL] - powershell.exe terminated or terminal UI was closed
02/04/2019 09:21:46 [NORMAL] - Language server startup failed.
02/04/2019 09:21:46 [ERROR] - The language service could not be started: 
02/04/2019 09:21:46 [ERROR] - Timed out waiting for session file to appear.
```

VScode version :
``` 
1.32.3
a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4
x64
```

Extensions : 

```
fatihacet.gitlab-workflow@2.0.1
ms-mssql.mssql@1.5.0
ms-vscode.powershell@1.12.0
shakram02.bash-beautify@0.1.1
```

PSversiontable : 

```
Name                           Value
----                           -----
PSVersion                      5.1.17763.316
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.316
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

Thanks a lot !

**Comment by toniino38** [comment]
I downgraded on 1.11.0 and everything's working well. 
There is a bug with 1.12.0 my colleague is in the same situation.

**Comment by SydneyhSmith** [comment]
@toniino38 thanks for reporting this, we are looking into the issue, Is the device you are using, restricted or locked down in anyway, for example with Group Policy?

**Comment by TylerLeonhardt** [comment]
To add to @SydneyhSmith's response... could it be an anti virus software? We've gotten similar issues: #1831

**Comment by toniino38** [comment]
We have no restrictions by GPO, however we have Bitdefender as antivirus software, I'll tried to get a laptop in the same configuration without antivirus installed to check it.

**Comment by toniino38** [comment]
I tried with and without BitDefender on a brand-new laptop and it works. I don't know why the extension doesn't work on our laptops :(

**Comment by rjmholt** [comment]
Looks like https://github.com/PowerShell/vscode-powershell/issues/1831, which has had a fix merged and will be in the next release.

Deduplicating this issue to https://github.com/PowerShell/vscode-powershell/issues/1831. If it turns out to be a different problem, we can reopen.

**Comment by hvCoder2k18** [comment]
I also had a similar issue last week. Besides, run selection would also not work. 

To resolve this, I tried reinstalling vscode, reinstalling powershell extension in vain.

Finally, went into the powershell extension settings and found the path for powershell.powerShellExePath to be empty. Once I defined a path to powershell.exe there everything started working again

![image](https://user-images.githubusercontent.com/29419322/69747903-d2ac2c00-110c-11ea-8930-b0f1abd3ab57.png)
